useful-scripts/sql/FootnoteData.sql

```diff
@@ -442,29 +582,104 @@ EXEC InsertFilterItemFootnote @subject_absence_by_characteristic, @filterItem_Et
+
+EXEC InsertFilterItemFootnote @subject_exclusions_by_characteristic, @filterItem_ethnicity_unclassified, @footnote_id_34;
+EXEC InsertFilterItemFootnote @subject_exclusions_by_characteristic, @filterItem_ethnicity_unclassified_male, @footnote_id_34;
+EXEC InsertFilterItemFootnote @subject_exclusions_by_characteristic, @filterItem_ethnicity_unclassified_female, @footnote_id_34;
+
+EXEC InsertFilterItemFootnote @subject_duration_of_fixed_exclusions, @filterItem_State_funded_primary, @footnote_id_38;
+EXEC InsertFilterItemFootnote @subject_exclusions_by_characteristic, @filterItem_State_funded_primary, @footnote_id_38;
+EXEC InsertFilterItemFootnote @subject_exclusions_by_geographic_level, @filterItem_State_funded_primary, @footnote_id_38;
+EXEC InsertFilterItemFootnote @subject_exclusions_by_reason, @filterItem_State_funded_primary, @footnote_id_38;
+EXEC InsertFilterItemFootnote @subject_number_of_fixed_exclusions, @filterItem_State_funded_primary, @footnote_id_38;
+EXEC InsertFilterItemFootnote @subject_total_days_missed_due_to_fixed_period_exclusions, @filterItem_State_funded_primary, @footnote_id_38;
+
+EXEC InsertFilterItemFootnote @subject_duration_of_fixed_exclusions, @filterItem_State_funded_secondary, @footnote_id_39;
+EXEC InsertFilterItemFootnote @subject_exclusions_by_characteristic, @filterItem_State_funded_secondary, @footnote_id_39;
+EXEC InsertFilterItemFootnote @subject_exclusions_by_geographic_level, @filterItem_State_funded_secondary, @footnote_id_39;
+EXEC InsertFilterItemFootnote @subject_exclusions_by_reason, @filterItem_State_funded_secondary, @footnote_id_39;
+EXEC InsertFilterItemFootnote @subject_number_of_fixed_exclusions, @filterItem_State_funded_secondary, @footnote_id_39;
+EXEC InsertFilterItemFootnote @subject_total_days_missed_due_to_fixed_period_exclusions, @filterItem_State_funded_secondary, @footnote_id_39;
+
+EXEC InsertFilterItemFootnote @subject_duration_of_fixed_exclusions, @filterItem_Special, @footnote_id_40;
+EXEC InsertFilterItemFootnote @subject_exclusions_by_characteristic, @filterItem_Special, @footnote_id_40;
+EXEC InsertFilterItemFootnote @subject_exclusions_by_geographic_level, @filterItem_Special, @footnote_id_40;
+EXEC InsertFilterItemFootnote @subject_exclusions_by_reason, @filterItem_Special, @footnote_id_40;
+EXEC InsertFilterItemFootnote @subject_number_of_fixed_exclusions, @filterItem_Special, @footnote_id_40;
+EXEC InsertFilterItemFootnote @subject_total_days_missed_due_to_fixed_period_exclusions, @filterItem_Special, @footnote_id_40;
+
 
 --
 -- Filters
 --
 
 
-DECLARE @filter_Characteristic NVARCHAR(max) = 'Characteristic';
-
-EXEC InsertFilterFootnote @subject_absence_by_characteristic, @filter_Characteristic, @footnote_id_15;
-EXEC InsertFilterFootnote @subject_absence_by_characteristic, @filter_Characteristic, @footnote_id_16;
+-- DECLARE @filter_Characteristic NVARCHAR(max) = 'Characteristic';
```

Comment:
You can remove this section now that you are applying the footnote by the Characteristic filter groups.

Reply:
I've removed it. My thinking in leaving it was it would provide an example case if someone wanted to attach a footnote to a filter group in the future.